src/app/models/event.model.ts

```diff
@@ -228,7 +230,19 @@ export class EventQuery {
                     )
                 };
 
+              case 'workitemtypes':
+                return {
+                  ...event,
+                  modifier: this.userQuery.getUserObservableById(event.modifierId),
+                  newValueRelationshipsObs: event.newValueRelationships.map(item => {
+                    return this.workitemTypeQuery.getWorkItemTypeById(item.id);
+                  }),
+                  oldValueRelationshipsObs: event.oldValueRelationships.map(item => {
+                    return this.workitemTypeQuery.getWorkItemTypeById(item.id);
+                  })
+                };
               default:
+                console.log('Unknown event: ', event);
```

Comment:
Please comment here why this console.log is needed.

Reply:
What? This console log is self-explanatory. What else do you think should be added?

Reply:
Just a comment saying don't remove this console and it logs the unknown event.

Reply:
Done.